gradle/versions.gradle

```diff
@@ -3,15 +3,18 @@ ext {
     // SDK
     minSdkVersion = 16
     automationAppMinSDKVersion = 21
-    targetSdkVersion = 27
+    targetSdkVersion = 28
     compileSdkVersion = 28
     buildToolsVersion = "28.0.3"
 
     // Plugins
-    gradleVersion = '3.3.1'
+    gradleVersion = '4.0.1'
```

Comment:
Is this the Gradle version or the Gradle plugin version?  The Gradle version should be 6.1.1, right?

Reply:
gradle version